src/Orleans.Core/Shims/RuntimeStatisticsGroup.cs

```diff
@@ -1,34 +1,22 @@
-﻿using System;
+using System;
 using Microsoft.Extensions.Logging;
+using Orleans.Statistics;
 
 namespace Orleans.Runtime
 {
-    internal class RuntimeStatisticsGroup : IDisposable
+    internal class DummyHostEnvironmentStatistics : IHostEnvironmentStatistics
     {
-        private readonly ILogger logger;
-        public long MemoryUsage => 0;
-        public long TotalPhysicalMemory => int.MaxValue;
-        public long AvailableMemory => TotalPhysicalMemory;
-        public float CpuUsage => 0;
+        public long? TotalPhysicalMemory => null;
 
-        public RuntimeStatisticsGroup(ILoggerFactory loggerFactory)
-        {
-            this.logger = loggerFactory.CreateLogger<RuntimeStatisticsGroup>();
-        }
-
-        internal void Start()
-        {
-
-            logger.Warn(ErrorCode.PerfCounterNotRegistered,
-                "CPU & Memory perf counters are not available in .NET Standard. Load shedding will not work yet");
-        }
+        public float? CpuUsage => null;
```

Comment:
I think they are null by default , you don't need to assign them

Reply:
- I think it is bad to rely on "default" value
- Here I defined a getter, I have to return something